app/res/drawable/progressbar_modern.xml

```diff
@@ -2,20 +2,22 @@
 <layer-list xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:id="@android:id/background">
         <shape>
+            <corners android:radius="2dp" />
```

Comment:
@pm-dimagi this one too should match the progress bar

Reply:
done